src/lib/containers/access_requirements/AddConditionsForUseButton.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from 'react'
+import { Button } from '@mui/material'
+import IconSvg from '../IconSvg'
+import ImposeRestrictionDialog from './ImposeRestrictionDialog'
+import { useGetEntityBundle } from '../../utils/hooks/SynapseAPI'
+import { RestrictionLevel } from '../../utils/synapseTypes'
+import {
+  isEntityRefCollectionView,
+  isEntityView,
+} from '../../utils/functions/EntityTypeUtils'
+import { useIsCurrentUserACTMember } from '../../utils/hooks/SynapseAPI/team/useTeamMembers'
+
+export type AddConditionsForUseButtonProps = {
+  entityId: string
+  /* Handler for when an ACT member clicks the button. */
+  onACTMemberClick: () => void
```

Comment:
In SWC, if ACT clicks this button, it goes straight to an ACT-only page, instead of showing the dialog.